src/status_im/infra/events.cljs

```diff
@@ -0,0 +1,25 @@
+(ns status-im.infra.events
+  (:require
+    [utils.re-frame :as rf]))
+
+;; Log error message and call error handle event if any
+(rf/reg-event-fx :infra.rpc/log-error
+ (fn [_ [{:keys [method params error-handler-event]} error]]
+   (merge
```

Comment:
The handler should use `:fx` effect only according to guidelines, then `merge` won't be needed.